doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_gs.rst

```diff
@@ -46,27 +46,8 @@ You also need the following:
 
 * `Git`_ or `Git for Windows`_ (on Linux and Mac, or Windows, respectively)
 * `curl`_
-* SEGGER J-Link |jlink_ver| and, on Windows, also the SEGGER USB Driver for J-Link from `SEGGER J-Link`_ |jlink_ver|
-
-   .. note::
-      To install the SEGGER USB Driver for J-Link on Windows, manually install J-Link |jlink_ver| from the command line using the ``-InstUSBDriver=1`` parameter:
-
-      1. Navigate to the download location of the J-Link executable and run one of the following commands:
-
-         * From the Command Prompt::
-
-            JLink_Windows_V794i_x86_64.exe -InstUSBDriver=1
-
-         * From PowerShell::
-
-            .\JLink_Windows_V794i_x86_64.exe -InstUSBDriver=1
-
-      #. Follow the on-screen instructions.
-      #. After installing, ensure the J-Link executable can be run from anywhere on your system:
-
-         * For Linux and MacOS, add it to the system path.
-         * For Windows, add it to the environment variables.
-
+* SEGGER J-Link |jlink_ver| and, on Windows, also the SEGGER USB Driver for J-Link from `SEGGER J-Link`_ |jlink_ver|.
+  For information how to install the USB Driver, see the `nRF Util prerequisites`_ documentation.
```

Comment:
```suggestion
  For information on how to install the USB Driver, see the `nRF Util prequisites <nRF Util prerequisites_>`_ documentation.
```

Reply:
I won't be updating `prerequisites` to `prequisites`.

Reply:
Sorry for the typo. It is gone now.